### PR TITLE
Simplified home page text

### LINKED ISSUE #241: Proposal: improve the user experience on the front page
I personally loathe the Wall Of Text (WOT) on the front page:

screen shot 2016-03-31 at 4 32 14 pm

I vote for either:
1. Showing only the 1st line and allowing the curious user see the full WOT with a "more" button that expands in place
2. Replacing this with a concise mission statement or similar, with the WOT available on a separate About page
3. Redesigning the front page to be a modern text-light splash screen with clear links to obtain the various pieces of information currently in the WOT

3 would obviously take a bit more effort

Indicate preference below, with `0` for the status quo, let's say `(-10,10)`, e.g
1. -10
2. +2
3. +2
4. +10

### PR DESCRIPTION
**Pull request by nlharris**
Fixes #241 

This is a starting point--I'm sure it will need tweaks in formatting, and perhaps improvement in text as well. But I can't see what it will look like until it's merged.

### CODE REVIEW

File: index.html
Comment (matentzn) [inline index.html:12]: Has this subheading been approved by someone other than us?

Comment (matentzn) [reply index.html:12]: @nlharris

**Comment by matentzn** [comment]
I like this, but would like @jamesaoverton to look at this before merging. Maybe next week.

File: index.html
Comment (nlharris) [reply index.html:12]: The proposed new home page content was discussed during the last OBO Operations meeting and there were some suggestions on it (which I responded to) but no one specifically commented on that text (which I believe was @mellybelly's invention). The general consensus was that the overall revisions looked good, and that I should go ahead and make a PR and we could continue to refine the wording. I welcome feedback!

**Review by nicolevasilevsky — APPROVED** [review]

File: index.html
Comment (nlharris) [inline index.html:29]: might be how-do-i-register-my-ontology.html (that page name has been switched back, I think)

Comment (nicolevasilevsky) [reply index.html:29]: yes, should be register

Comment (nicolevasilevsky) [reply index.html:29]: I think this page needs to be reverted back to say register:
https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/faq/how-do-i-submit-my-ontology.md

(I thought I did this? But I guess not)

Comment (nlharris) [reply index.html:29]: i thought you did too, that's why i'm so confused! i guess it somehow got lost in the shuffle? do you want to (re)rename it or should i?

Comment (nicolevasilevsky) [reply index.html:29]: I can do it!

Comment (nicolevasilevsky) [reply index.html:29]: I did it on this same branch

Comment (nicolevasilevsky) [reply index.html]: I fixed it above too (on line 29). Hence, now it says outdated.

Comment (nlharris) [reply index.html]: Thanks for fixing this link. What I meant is are you going to actually rename the file how-do-i-submit to how-do-i-register? because -register still 404s. http://obofoundry.org/faq/how-do-i-register-my-ontology.html

Comment (nicolevasilevsky) [reply index.html]: I did rename the file, you can see it on the files changed above:
image

**Review by matentzn — APPROVED** [review]

**Review by wdduncan — APPROVED** [review]

**Comment by nlharris** [comment]
Thanks, all, for the approvals. I am waiting for approval from @jamesaoverton before I merge.

**Review by jamesaoverton — APPROVED** [review]
The content is good, and I didn't change it.

I tested this locally, fixed some links, removed the blue box, and cleaned up some code style.

**Comment by nlharris** [comment]
Thanks! Yeah, I couldn't really check the display style before it was merged. I appreciate your help!

**Pull request merged**
